modules/instances/main.tf

```diff
@@ -24,7 +24,7 @@ resource "aws_key_pair" "pe_adm" {
   public_key = file(var.ssh_key)
 }
 
-# Instances to run PE Master
+# PE MoM instances in xlarge and large or the only instance in standard
```

Comment:
Let's adopt the new PE architecture vocabulary:

MoM -> Master
Compile Master -> Compiler
Primary Master -> Master
Primary Master Replica -> Master Replica